pages/common/valgrind.md

```diff
@@ -0,0 +1,20 @@
+# valgrind
+
+> Wrapper for a set of expert tools for profiling, optimizing and debugging programs.
+> Commonly used tools include `memcheck`, `cachegrind`, `callgrind`, `massif`, `helgrind`, and `drd`.
+
+- Use the (default) Memcheck tool to show a diagnostic of memory usage by `program`:
```

Comment:
I think it makes sense to capitalize proper names in the descriptions, and use lowercase for the actual invocations in the commands.